pkg/internal/apis/certmanager/validation/issuer.go

```diff
@@ -142,6 +150,12 @@ func ValidateACMEIssuerChallengeSolverConfig(sol *cmacme.ACMEChallengeSolver, fl
 		el = append(el, field.Required(fldPath, "no solver type configured"))
 	}
 
+	if sol.Selector != nil {
+		if len(sol.Selector.DNSNames) == 0 && len(sol.Selector.DNSZones) == 0 && len(sol.Selector.MatchLabels) == 0 {
+			el = append(el, field.Required(fldPath.Child("selector"), "selector need at least 1 item in dnsNames, dnsZones or matchLabels"))
```

Comment:
```suggestion
			el = append(el, field.Required(fldPath.Child("selector"), "need at least 1 item in dnsNames, dnsZones or matchLabels"))
```

Nit, the field name will already be in the error prefix by virtue of the fldPath